docker/Dockerfile

```diff
@@ -5,7 +5,9 @@
 #
 # SPDX-License-Identifier: LGPL-3.0-or-later
 
-FROM ubuntu:noble
+ARG UBUNTU_VERSION=20.04
+FROM ubuntu:${UBUNTU_VERSION}
+ARG UBUNTU_VERSION=20.04
 LABEL org.opencontainers.image.description="Image containing all the dependencies required for building and testing 4C"
```

Comment:
Why does the `UBUNTU_VERSION` arg appear twice?

Reply:
ARGs are cleared after FROM. But you are rightm we don't need `UBUNTU_VERSION` in this particular case afterwards, so we can remove it here.